app/src/components/vis/vis-network-directive.js

```diff
@@ -16,7 +16,6 @@ function visNetwork(visNetworkOptions, AnnyFactory, $rootScope) {
             var input = neuron.input.toFixed(3)
             var output = neuron.output.toFixed(3)
             var delta = neuron.delta.toFixed(6)
-            var error = neuron.error.toFixed(3)
```

Comment:
Since we use vanilla backprop now, we have no need for the error (only the delta).